schema.sql

```diff
@@ -9,4 +9,23 @@ CREATE TABLE animals(
    weight_kg         REAL
 );
 
-ALTER TABLE animals ADD species CHAR(100);
\ No newline at end of file
+ALTER TABLE animals ADD species CHAR(100);
+
+CREATE TABLE owners(
+   id  SERIAL PRIMARY KEY     NOT NULL,
+   full_name           TEXT    NOT NULL,
```

Comment:
- [ ] [Optional] full_name should be a string (TEXT should work but string recommended)